.github/workflows/release-changelog.yml

```diff
@@ -0,0 +1,49 @@
+name: Changelog vs Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@main
+        with:
+          configuration: ".github/changelog-configuration.json"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install JDK ${{ matrix.java_version }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.1.0
+
+      - name: Build project
+        run: ./gradlew clean build
+        env:
+          VERSION: ${{ github.ref }}
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
```

Comment:
What's about to use this ?

      - name: Get the version
        id: tagger
        uses: jimschubert/query-tag-action@v2
        with:
          skip-unshallow: 'true'
          abbrev: false
          commit-ish: HEAD

and use it with `${{steps.tagger.outputs.tag}}`

Reply:
Up to you. Propose to do it in different PR - don't want to lose time with testing. If you don't mind.

Reply:
Ah wait, it is just easier for me to test it now. Let me try it.

Reply:
This is currently missing

https://github.com/MikeOrtiz/TouchImageView/pull/337#discussion_r602863105